programs/mango-v4/src/state/bank.rs

```diff
@@ -204,6 +204,10 @@ pub struct Bank {
     /// zero means none, in token native
     pub deposit_limit: u64,
 
+    /// Largest amount of tokens that might be added the the bank based on
+    /// oenbook open order execution.
+    pub potential_openbook_tokens: u64,
```

Comment:
it's too dangerous to add new fields that are not at the end, even after merges. Please move this field to the end, just before `reserved`